test/Microsoft.SqlTools.ServiceLayer.UnitTests/Microsoft.SqlTools.ServiceLayer.UnitTests.csproj

```diff
@@ -5,7 +5,8 @@
 		<DefineConstants>$(DefineConstants);NETCOREAPP1_0;TRACE</DefineConstants>
 		<IsPackable>false</IsPackable>
 		<ApplicationIcon />
-		<StartupObject />		
+		<StartupObject />
+		<NoWarn>$(NoWarn);SYSLIB1045;IDE0200;IDE0230;CA1311;CA1852;CA1854</NoWarn>
```

Comment:
There are some suppressed warnings, as they impact build.
We should address them area-wise soon.